lib/add-to-org/views/success.erb

```diff
@@ -6,7 +6,7 @@
   </head>
   <body>
     <h1>You're all set</h1>
-    <p class="lead">You simply need to <a href="https://github.com/orgs/government/invitation?return_to=<%= ERB::Util.html_escape(redirect) %>">confirm your invitation to join the organization</a> on GitHub.</p>
+    <p class="lead">You simply need to <a href="https://github.com/orgs/<%= ENV['GITHUB_ORG_ID'] %>/invitation?return_to=<%= ERB::Util.html_escape(redirect) %>">confirm your invitation to join the organization</a> on GitHub.</p>
```

Comment:
I wasn't sure if I could also refer to the `org_id` method here somehow, but I guess it doesn't matter really.